### PR TITLE
implement areaOfTriangle.js and add phi.js, phi.test.js

### LINKED ISSUE #334: Fix areaOfTriangle method
There is currently a method called `areaOfTriangle` that lives in `/lib/areaOfTriangle.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test areaOfTriangle` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by stillNovice**
Closes #334 

- [] Running `yarn lint` does not trigger any linter errors
- [] The test of the method you have fixed is passing
- [] You have written a new skeleton method for someone else to work on!
- [] You have written tests to accompany your skeleton method

### CODE REVIEW

File: test/phi.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const { phi } = require("../lib");
+
+describe("phi", () => {
+  test("phi(1) equals 1", () => {
```
Comment (BlakeGuilloud) [inline test/phi.test.js]: should this be "phi(7) equals 1"?

Comment (stillNovice) [reply test/phi.test.js]: No it should be phi(1), thanks for pointing this out. I'll fix this

**Comment by ktilcu** [comment]
Thanks for the contributions. You are one commit closer to a T-shirt!
